src/lang/en-US/Workflows.ts

```diff
@@ -248,6 +245,10 @@ export const Workflows: Record<string, string> = {
     'fieldSelection.table.label.unsatisfiable': `Unsatisfiable`,
     'fieldSelection.table.label.unknown': `Unknown`,
     'fieldSelection.table.label.filter': `Filter fields`,
+    'fieldSelection.massActionMenu.recommended.label': `Default`,
+    'fieldSelection.massActionMenu.recommended.description': `Allow the connector to dynamically select fields`,
+    'fieldSelection.massActionMenu.excludeAll.label': `Exclude All`,
+    'fieldSelection.massActionMenu.excludeAll.description': `Exclude non-essential fields`,
```

Comment:
I could go either way - but it does seem a bit weird for one option to have `All` in the label and the other does not. But something like `Default All` is _kinda_ odd.